src/views/home/index.tsx

```diff
@@ -41,12 +41,18 @@ export const HomeView = () => {
     const ghToken = Cookies.get('gh_token');
 
     async function getAccessToken() {
-      const code = gh.parseOAuthCode() || null;      
+      const code = gh.parseOAuthCode() || null;
 
       if (code) {
         const { data } = await axios.get(`http://localhost:4000/auth?code=${code}`);
         if (data.success && !isEmpty(data.data.token)) {
           Cookies.set('gh_token', data.data.token)
+
+          // const userRes = await axios.get(`https://api.github.com/user`, { headers: new Headers({ "Authorization": `token ${data.data.token}` }) });
+
+          // if (userRes.data.success && !isEmpty(userRes.data)) {
+          //   setUserInfo(userRes.data)
```

Comment:
sorry why is this commented? im using it on lines 71 onwards

Reply:
maybe im missing it but `setUserInfo` was never called. thats why i was trying to figure it out.

Reply:
oh isee, is this from a conflict? yea u can remove that. i used `setUserInfo` in `getUserInfo` function

Reply:
any shot you can share a permalink to that? idk i cant find it.

Reply:
This snippet was started because i noticed that after clicking Connect to GitHub my user info wasnt showing up. So I noticed the state wasnt being set.

Reply:
https://github.com/repomint/repomint-web/blob/966130ec97bd35677a50831457de4257f0cfd9a8/src/views/home/index.tsx#L79

Reply:
woah its not in my local. what did i do haha. sec

Reply:
ah crap i thought #33 was merged